src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    public String value;
```

Comment:
Are there ways we can prevent the exposing of `value`? Perhaps using the `protected` scope as `isPrivate`? =P